ox-hugo.el

```diff
@@ -126,7 +126,11 @@ directory where all Hugo posts should go by default."
                    (:hugo-section "HUGO_SECTION" nil org-hugo-default-section-directory)
                    (:hugo-base-dir "HUGO_BASE_DIR" nil nil)
                    (:hugo-static-images "HUGO_STATIC_IMAGES" nil "images")
-                   (:hugo-code-fence "HUGO_CODE_FENCE" nil "t")))
+                   (:hugo-code-fence "HUGO_CODE_FENCE" nil "t")
+                   (:hugo-static-images "HUGO_STATIC_IMAGES" "images" nil)
+                   (:hugo-menu-name "HUGO_MENU_NAME" "main" nil)
+                   (:hugo-menu-weight "HUGO_MENU_WEIGHT" 20 nil)
```

Comment:
Same swapping of `20` and `nil` here. I'd be surprised if the default value of menu weight showed up to be 20 for you.